lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -232,6 +230,7 @@ import Shelley.Spec.Ledger.BaseTypes
 import Type.Reflection
     ( Typeable, typeRep )
 
+import qualified Cardano.Address.Style.Shelley as SH
```

Comment:
I would have preferred not changing it to use qualified imports. This module is called `...Shelley.Compatibility` so a prefix like `SH` (I assume it stands for Shelley?) is redundant.